docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+s# HVAC_SystemZoneAssignment - Rule 11-10
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** "The service water heating system type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2"  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each ServiceWaterHeatingEquipment  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW Equipment is applicable
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_equip_dict = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)`
+- look through each of the SHW bats in the building: `for shw_bat in shw_bats_and_equip_dict:`
+  - look through each SHW Equipment: `for service_water_heating_equipment_id in shw_bats_and_equip_dict[shw_bat]: CONTINUE TO RULE LOGIC`
```

Comment:
every service water heating equipment is applicable, so we need to evaluate the rule for each one.  I'm etting to the SWH equipment this way because it's an efficienct way to also have the shw_bat

Reply:
@KarenWGard The comment is not about the way you get to it. It is about making it clear that the rule is NOT_APPLICABLE when there is no SWH equipment. Do you follow me?

Reply:
the evaluation context is each ServiceWaterHeatingEquipment - I updated the language for more clarity, but every single ServiceWaterHeatingEquipment will be evaluated